.github/workflows/cibuild.yml

```diff
@@ -54,10 +55,40 @@ jobs:
     - name: Python 3.8 with astropy32
       if: matrix.python-version == 3.8
       run: tox -e py38-test-astropy32
+    - name: Python 3.8 with astropy40
```

Comment:
I guess this works. `astropy` has a different style if you are interested: https://github.com/astropy/astropy/blob/master/.github/workflows/ci_workflows.yml

Reply:
Yes, I will do that if we agree to a regression matrix that is symmetrical.

Reply:
@pllim do you know if `astropy` releases minor versions to work with newer Python version? E.g. recently released P3.9 expected to work with latest `astropy4.0`?

Reply:
astropy 4.0.4 should work with Python 3.9; I see the wheels for Python 3.9 at https://pypi.org/project/astropy/4.0.4/#files